086-archive-canary.md

```diff
@@ -0,0 +1,94 @@
+Additionally, we mentioned that in case that community finds Canary useful, provides information about what can be improved (in terms of metrics, functionality), and we will have enough
+capacity and resources to maintain it, we can open a new proposal for writing the Canary vol.2 in Java.
+
+## Proposal
+
+After this proposal is approved, we should:
+- archive the Canary project
+- remove the Canary install files from Strimzi operators repository 
+  - now from the `/packaging/install` folder
+  - as part of the next Strimzi release, the installation files will be removed from the `/install` folder
+- delete the mentions about the Canary project from the documentation and automation files (Makefiles)
+- inform users about archiving the project
+
+In case that community: 
+
+- finds Canary useful 
+- will be able to provide additional information about what metrics or functionality can be improved or used
+- we will be able to maintain the project
+
+we can write a new proposal for Canary vol.2 in Java.
+
+## Affected/not affected projects
+
+The only affected project is the [Canary](https://github.com/strimzi/strimzi-canary) that should be archived.
+In the Strimzi operators repository, the only affected parts are the [installation files](https://github.com/strimzi/strimzi-kafka-operator/tree/main/install/canary) and 
+[in development installation files](https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/install/canary) that should be deleted by this proposal.
+
+## Compatibility
+
+The backwards compatibility is not relevant in this case, as the project will be archived and there is no other solution currently that would replace it.
+
+## Rejected alternatives
+
+### Maintaining Canary in Go and providing additional functionality
+
+One of the rejected alternative was to keep updating the Canary in Go and add more functionality to it.
+This was discussed and rejected because of:
+- lack of Go experts in the Strimzi organization
+- resources - it would take a lot of time to learn Go and properly testing every new change without knowing how it will work
+- missing functionality in the Sarama Kafka client
+
+### Maintaining Canary for dependency updates
+
+Another one alternative was to keep Canary project and updating the dependencies.
+Some of the dependency updates can be without breaking changes, but from time to time there are changes in the dependencies that requires additional 
+changes to the code, which brings us to the same situation as the previous alternative - someone would need to do the changes to the code, 
+test it properly and then release it.
+Because of these issues, we decided to reject this alternative.
+
+### Rewrite Canary in Java
+
+Final alternative was to rewrite Canary in Java, but as was mentioned in the [Canary in Java](#canary-in-java), it would not be the same Canary as in Go, 
```

Comment:
```suggestion
A final alternative was to rewrite Canary in Java, but as was mentioned in the [Canary in Java](#canary-in-java), it would not be the same Canary as in Go, 
```